astropy/coordinates/name_resolve.py

```diff
@@ -33,6 +33,7 @@ class sesame_url(ScienceState):
     """
 
     _value = [
+        "https://cds.unistra.fr/cgi-bin/nph-sesame/",
```

Comment:
This is technically software behavior change, so should add change log?

Reply:
I'll leave it to your expertise on how to best word the change log content. The audience is end-user who might not know/care how things work behind the scenes. So how will this change affect them (directly or indirectly). A concise sentence is fine. Thanks!

Reply:
Is this supposed to replace the `cdsweb.u-strasbg.fr` URL ? If this is the same server I guess we should keep only one URL.

Reply:
> The audience is end-user who might not know/care how things work behind the scenes. So how will this change affect them (directly or indirectly). A concise sentence is fine. Thanks!

Then there is no change for the users at all. :)

> Is this supposed to replace the cdsweb.u-strasbg.fr URL ? If this is the same server I guess we should keep only one URL.

 u-strasbg.fr will be stopped one day. The new domain name is unistra.fr

Reply:
Then how about a change log that says something like, "Updated CDS URL for Sesame look-up as the old URL is deprecated"?

Also, since the old URL will stop some day, we should remove it here, as @saimn suggested above?